meta/main.yml

```diff
@@ -3,7 +3,8 @@ galaxy_info:
   description: "Candlepin Entitlement Management"
   company: "Red Hat"
   license: license (GPLv2)
-  min_ansible_version: 2.3
+  min_ansible_version: '2.4'
```

Comment:
our role works this far back but molecule only really works on 2.8 and 2.9 ... I recommend increasing this if all the devs are not using older ansibles.